src/app/public/testing/tabs/tab-fixture-tab.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Properties of a SKY UX tab.
+ */
+export interface SkyTabsetFixtureTab {
```

Comment:
I'm still not crazy about this name. Admittedly, I followed the same pattern but I'm also struggling with it.... 🤔

Reply:
I can live with it.